src/util/key.rs

```diff
@@ -75,47 +75,22 @@ use crate::cryptonote::hash;
 use serde::{Deserialize, Serialize};
 
 /// Errors that might occur during key decoding
-#[derive(Debug, PartialEq)]
+#[derive(Fail, Debug, PartialEq)]
 pub enum Error {
     /// Invalid input lenght
-    InvalidLenght,
+    #[fail(display = "invalid length")]
+    InvalidLength,
```

Comment:
Thanks for catching that mistake. Can you also correct it in comments and documentation

Reply:
No problem :') Will change all occurrences.